docs/Typedefs.md

```diff
@@ -24,6 +24,20 @@ A single audience used for authenticating and communicating with an individual A
 | apiToken    | <code>string</code> |
 | expiresAt   | <code>number</code> |
 
+<a name="AxiosInterceptor"></a>
+
+## AxiosInterceptor : <code>Object</code>
+
+An object of interceptors that get called on every request or response.
+More information at [axios Interceptors](https://github.com/axios/axios#interceptors)
+
+**Kind**: global typedef
+
+| Param                 | Type                  | Description                                                    |
+| --------------------- | --------------------- | -------------------------------------------------------------- |
+| interceptor.fulfilled | <code>function</code> | A function that is run on every successful request or response |
+| interceptor.rejected  | <code>function</code> | A function that is run on every failed request or response     |
+
```

Comment:
This is probably me not understanding the interceptors that well since I just skimmed the axios interceptor section in their docs but it seemed to me that the interceptors are bifurcated between requests and responses and not fulled and rejected? Hope my question makes sense. Like for the `rejected`, how would the interceptor know that the request was reject since it hasn't received a response yet?

Reply:
For the words here, I went with the verbiage axios uses in their code (which doesn't really differentiate between request or response: https://github.com/axios/axios/blob/master/lib/core/InterceptorManager.js#L17).

In the case of rejected requests, I believe those interceptors would be invoked if there was an error in a previous interceptor (and would still return .a promise)